### DISCUSSION TITLE
OpenAPI elements are not searchable

### DISCUSSION THREAD

**Discussion by robertmerten** [GitbookIO/community#582] [Feature Requests]
When using the gitbook [search endpoint](https://developer.gitbook.com:8443/gitbook-api/reference/search), content that is in OpenAPI blocks is not searched.

Is this intended behaviour or a missing feature / bug.

**Comment by aikoii** [comment]
Hey @robertmerten, you're right, searching within OpenAPI blocks isn't currently supported. It's a feature we're looking to add in the future 🙂

**Comment by robertmerten** [comment]
@aikoii Hey,

is there any update or timeline on this?

**Reply by aikoii** [reply]
Hey @robertmerten 👋, unfortunately, there's no update on that yet.
